src/app/(health-worker)/children/register/page.tsx

```
@@ -0,0 +1,521 @@
+        <p className="mt-1 text-sm text-gray-500">
+          Register a newborn and link to the mother&apos;s health record
+        </p>
+      </div>
+
+      <form onSubmit={handleSubmit(onSubmit)} className="space-y-5">
+        {/* Single flat white container with divider */}
+        <div className="border border-gray-200 bg-white rounded-lg">
+          {/* Mother search */}
+          <div className="p-6 border-b border-gray-200">
+            <h2 className="mb-4 text-base font-semibold text-gray-900">
+              Mother's health record
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)